packages/compiler/src/init/init.ts

```diff
@@ -142,24 +143,34 @@ async function isDirectoryEmpty(directory: string) {
   }
 }
 
+function warning(message: string) {
+  // eslint-disable-next-line no-console
+  console.log(pc.yellow(`warning: ${message}`));
+}
+function success(message: string) {
+  // eslint-disable-next-line no-console
+  console.log(pc.green(`success: ${message}`));
+}
+function whiteline() {
+  // eslint-disable-next-line no-console
+  console.log("");
+}
+
 async function confirmDirectoryEmpty(directory: string) {
   if (await isDirectoryEmpty(directory)) {
     return true;
   }
 
-  return confirm(
-    `Folder '${directory}' is not empty. Are you sure you want to initialize a new project here?`,
-  );
+  warning(`Folder '${pc.cyan(directory)}' is not empty.`);
```

Comment:
hi @timotheeguerin! Thoughts on removing  `''` encasing the folder path? For sighted users, the cyan color treatment should be enough. Updated the prototype too to reflect this:

![CleanShot 2025-02-20 at 13 39 51](https://github.com/user-attachments/assets/2ab9b6c1-7ba2-41fb-b9c1-c5f1f9bbd8cc)


```suggestion
  warning(`Folder ${pc.cyan(directory)} is not empty.`);
```

Reply:
yeah that sounds good